packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -139,18 +166,41 @@ function Directions({ isOpen, onBack, directions }) {
                         <label className="directions__label">
                             From
                         </label>
-                        {directions?.originLocation && <div>{directions.originLocation.properties.name}</div>}
+                        {directions?.originLocation &&
+                            <div className="directions__info">
+                                <div className="directions__content">
+                                    <div className='directions__name'>
+                                        {directions?.originLocation.properties.name}
+                                        {directions?.originLocation.properties.name !== 'My Position' && <div>·</div>}
+                                        <mi-location-info ref={originInfoElement} />
+                                    </div>
+                                </div>
+                            </div>
+                        }
                     </div>
                     <div className="directions__container">
                         <label className="directions__label">
                             To
                         </label>
-                        {directions?.destinationLocation && <div>{directions.destinationLocation.properties.name}</div>}
+                        {directions?.destinationLocation &&
+                            <div className="directions__info">
+                                {destinationDisplayRule && directions.originLocation.name !== 'My Position' &&
```

Comment:
I am not sure I understand why originLocation is considered in this part about the destinationLocation. Can you explain?

Reply:
Oh, that is a mistake. It should be destination location instead of origin. My bad, I have changed it now. 🫣